package.json

```diff
@@ -79,6 +79,7 @@
     "parse-static-imports": "^1.1.0",
     "pprof": "^3.2.0",
     "prom-client": "^14.1.1",
+    "redis": "^4.6.5",
```

Comment:
Can you remove this? We are not adding redis for this destination.

Reply:
Done